src/crypto.js

```diff
@@ -40,12 +40,10 @@ exports.getRandomValues = function (buffer) {
     view = new Uint8Array(buffer),
     i;
 
-  if (offset + size > buf.length) {
-    throw new Error("Insufficient Randomness Allocated.");
-  }
-
+  // Hack: just wrap around.
```

Comment:
i'd really like a design where it's hard to shoot yourself in the foot. Having to check something afterward to know that you ran out crypto and should re-run with a bigger buffer seems like it's too easy to not do that check.

A design I think would remain safe is a switch to put crypto into a 'trial run' mode, where it gives back zeros and counts how many bytes are requested. then you can run once to learn the buffer size, and then again with entropy allocated. would that work?